app/models/wallet.rb

```diff
@@ -52,6 +52,10 @@ def currency
   def limited_fee_types?
     allowed_fee_types.present?
   end
+
+  def limited_billable_metrics?
```

Comment:
I think the name isn't ideal.
I'd use `limited_to_billable_metrics?` or even `billable_metrics_limited?`.

```suggestion
  def billable_metrics_limited?
```